gammapy/maps/base.py

```diff
@@ -498,6 +499,121 @@ def upsample(self, factor, order=0, preserve_counts=True):
         """
         pass
 
+    def get_image_by_coord(self, coords, copy=True):
+        """Return spatial map at the given axis coordinates.
+
+        Parameters
+        ----------
+        coords : tuple or dict
+            Tuple should be ordered as (x_0, ..., x_n) where x_i are coordinates
+            for non-spatial dimensions of the map. Dict should specify the axis
+            names of the non-spatial axes such as {'axes0': x_0, ..., 'axesn': x_n}.
+        copy : bool
+            Whether to make a copy of the data.
+
+        Examples
+        --------
+
```

Comment:
I think you have to add a:
```
::
```
or a
```
.. code-block::
```
to make this appear as a code block in the docs, no?

Reply:
Done